apps/src/javalab/JavabuilderConnection.js

```diff
@@ -36,12 +45,16 @@ export default class JavabuilderConnection {
 
   establishWebsocketConnection(token) {
     let url = this.javabuilderUrl;
+    const optionsStr = $.param({options: this.options});
```

Comment:
Looks like we also have an npm package that handles this: `query-string`
We have some utils set up for it as well: https://github.com/code-dot-org/code-dot-org/blob/47be99d6cf7df2be746b592906f50c0f3860b80a/apps/src/code-studio/utils.js
I seem to remember Star Labs was generally trying to move away from jquery syntax in favor of other options.